_layouts/home.html

```diff
@@ -0,0 +1,21 @@
+{% include top.html %}
+
+<body class="wrap">
+  <header>
+    {% include header.html %}
+  </header>
+  <div class="big-arrow-bg">
+    <div class="container p-lg-4 centered">
+      <img src="{{ site.baseurl }}/img/arrow-inverse.png" style="max-width: 80%;"/>
```

Comment:
I think the logo is currently too big. It's taking much of the window's real estate. Reduce it to e.g. 40%?

Reply:
We can experiment; this also seems like the kind of thing to get a real designer's opinion on.